Client/CloudAppClient.php

```diff
@@ -36,6 +36,14 @@ public function upload($archive)
     /**
      * {@inheritdoc}
      */
+    public function download()
+    {
+        throw new \LogicException(sprintf('Download method for %s is not implemented.', __CLASS__));
```

Comment:
I see it as a short speed bump until its implemented. If adding a new interface, how would you handle this https://github.com/dizda/CloudBackupBundle/blob/master/Manager/ClientManager.php#L40?